dts/bindings/base/pm.yaml

```diff
@@ -30,3 +30,9 @@ properties:
     description: |
       Automatically configure the device for runtime power management after the
       init function runs.
+
+  reinit-power-states:
+    type: phandles
+    description: |
+      System power states in which a peripheral's configuration is lost, and
+      must be reinitialized to ensure correct operation.
```

Comment:
this is something that is known by the system, ie, driver, so why put it in DT?

Reply:
This is different between every SoC series, e.g. SPI loses context in STOP0 on a STM32WB, but on STM32WL it's not until STOP2.

Reply:
The initial assumption was that devices state would be related with a power state but in practice it is fragile and since substates  addition they are even more de-coupled ... Basically the only relationship between them now is which power state will trigger devices suspensions ... that is reasonable for me.

Reply:
If we're fine with this proposal, I'd proposal to move this  and related commits in a dedicated PR

Reply:
Please, do this.

Reply:
Please see #60369